### ISSUE TITLE
Get-Package -updates lists packages that don't show up either under installed or updates tab

### ISSUE THREAD

**Issue by karann-msft** [NuGet/Home#6085]
The following packages show up when `Get-Package -updates` is run from the PMC. However, they are not listed either under installed or updates tab.

Microsoft.CodeDom.Providers.DotNetCompilerPlatform
Microsoft.Net.Compilers

![image](https://user-images.githubusercontent.com/16904420/31925196-f0bd282e-b83b-11e7-80b0-60398a32e97b.png)

Repro:
1. FIle->New->ASP.NET project
2. Open PMC and run  `Get-Package -updates`
3. Observe the output and compare with what you see under the updates tab under the PMUI

Expected:
The results are consistent.

Actual:
The PMC returns packages not found under the PMUI updates tab.

27017.0.d15rel

**Comment by mishra14** [comment]
@karann-msft what is the exact template that you are using?

**Comment by rrelyea** [comment]
Sounds like we need to fix get-package -updates to be like our PM UI.
Perhaps we update the UI to understand transitive dependencies and how we should list installed and updated ones...but we didn't make updates to PMC.

@jainaashish

**Comment by jainaashish** [comment]
@karann-msft PMC shows packages from 2 projects (webapplication1 and classLibrary1) where as PM UI is at project level. could that be the difference?

**Comment by karann-msft** [comment]
@jainaashish no, that's not it. The packages that the PMC is reporting, do not show up in either project as updates, or even installed for that matter, when viewed using PMUI.
